services/idp/pkg/config/config.go

```diff
@@ -112,6 +113,7 @@ type Settings struct {
 
 	CookieBackendURI string
 	CookieNames      []string
+	CookieSameSite   http.SameSite
```

Comment:
IMHO we should have an env-var & yaml annotation to configure that.

Reply:
i think no, since the idp is meant for internal use only... am i right?

Reply:
but on the same site.... im not against it... 🤣

Reply:
if you decide to have it, can u take over and add it? i expect load testing will consume most of my day

Reply:
wait yes you are right... this does not affect external idp configurations afaik....

Reply:
unless we provide the ability to authenticate extenal apps via the internal idp this is not needed....